packages/web-frontend/src/screens/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,135 @@
+import { useAuth } from './useAuth';
+import { useNavigation } from './useNavigation';
+import { ProjectCardList } from '../../containers/OverlayDiv/components/ProjectPage/ProjectCardList';
+import { useCustomLandingPages } from './useCustomLandingPages';
+
+const Wrapper = styled.div`
+  position: relative;
+  background-size: cover;
+  background-position: center;
+`;
+
+const Container = styled(MuiContainer)`
+  position: relative;
+  min-height: calc(100vh - 75px);
+  background-color: #bfbfbf;
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-top: 5%;
+`;
+
+const Projects = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  column-gap: 50px;
+  margin: 0 auto;
+  max-width: 1200px;
+  padding-top: 10%;
+`;
+
+const Title = styled(Typography)`
+  font-style: normal;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 3rem;
+  margin-bottom: 1.8rem;
+  color: white;
+  text-shadow: 1px 1px #333;
+  max-width: 480px;
+`;
+
+const NavBar = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background: #0f3150;
+  color: white;
+  height: 75px;
+  padding: 0 30px;
+
+  button {
+    margin-right: 15px;
+  }
+`;
+
+const Footer = styled.div`
+  position: absolute;
+  bottom: 0;
+  left: 50px;
+  right: 50px;
+  border-top: 1px solid white;
+  height: 75px;
+`;
+
+export const LandingPage = () => {
+  const {
+    navigateToProject,
+    navigateToRequestProjectAccess,
+    navigateToLogin,
+    navigateToLogout,
+  } = useNavigation();
+  const { projects } = useCustomLandingPages();
+  const { isUserLoggedIn, currentUserUsername } = useAuth();
+
+  return (
+    <>
+      <NavBar>
+        {currentUserUsername && (
+          <Box mr={3}>
+            {currentUserUsername}
+            <span style={{ marginLeft: 30 }}>|</span>
+          </Box>
+        )}
+        {isUserLoggedIn ? (
+          <Button onClick={navigateToLogout}>Logout</Button>
+        ) : (
+          <div>
+            <Button onClick={navigateToLogin}>Register</Button>
+            <Button onClick={navigateToLogin}>Login</Button>
+          </div>
+        )}
+      </NavBar>
+      <Wrapper>
+        <Container maxWidth={false}>
+          <Title variant="h1">Select a project below to view.</Title>
+          <Projects>
+            <ProjectCardList
+              projects={projects}
+              accessType
+              action={navigateToProject}
+              actionText="View project"
+            />
+            <ProjectCardList
+              projects={projects}
+              accessType="pending"
+              action={navigateToRequestProjectAccess}
+              actionText="Approval in progress"
+            />
+            <ProjectCardList
+              projects={projects}
+              accessType={false}
+              action={isUserLoggedIn ? navigateToRequestProjectAccess : navigateToLogin}
+              actionText={isUserLoggedIn ? 'Request access' : 'Log in'}
+            />
```

Comment:
I love the reusable component for `ProjectCardList`. One thought, which may or may not be useful, is that it looks like wherever `ProjectCardList` gets used it displays all three access types, with the same text, is that right? If so, would it make sense to make `ProjectCardList` just accept `projects` and potentially `actions` as an object keyed by the different types (or some similar way of passing in actions), and then `ProjectCardList` could loop through the projects and display accordingly?